index.html

```diff
@@ -31,10 +36,10 @@
     </header>
 
     <main>
-      <section class="container bg-white">
+      <section class="container">
```

Comment:
- [ ] Kindly make sure that your header does not have any static/fixed widths, just as the last reviewer suggested, please see video below to see how your desktop design looks

https://user-images.githubusercontent.com/69638013/178113631-1a73b6dd-c7dc-426a-86ee-e25d6fb5cff9.mp4

Reply:
- [ ] [optiona] @Alusp great work implementing the changes, it would also be nice ensuring in a desktop mode that the nav items are aligned to the right and not centered.